src/blueprint.zig

```diff
@@ -372,6 +374,246 @@ pub const Pattern = struct {
 	}
 };
 
+const BlockLike = union(enum) {
+	block: Block,
+	blockType: u16,
+};
+
+fn generatePropertyEnum() type {
+	var tempFields: [@typeInfo(Block).@"struct".decls.len]std.builtin.Type.EnumField = undefined;
+	var count = 0;
+
+	for(std.meta.declarations(Block)) |decl| {
+		const declInfo = @typeInfo(@TypeOf(@field(Block, decl.name)));
+		if(declInfo != .@"fn") continue;
+		if(declInfo.@"fn".return_type != bool) continue;
+		if(declInfo.@"fn".params.len > 1) continue;
```

Comment:
More correct would be `!= 1` since you probably don't want to catch 0-argument functions either.